doc/modules/svm.rst

```diff
@@ -339,7 +339,6 @@ floating point values instead of integer values::
     >>> regr.predict([[1, 1]])
     array([1.5])
 
-
```

Comment:
Let’s keep this as a blank line as in the [original document](https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/svm/_classes.py), and see if the CI tests pass.